docs/concepts/endpoint.md

```diff
@@ -0,0 +1,38 @@
+---
+id: endpoint
+title: Endpoint
```

Comment:
The content under this page needs a lot more attention. I think it does not reflect the exact point of the `Endpoint` concept.

Reply:
okay

Reply:
What is your plan for the concept section? Are you going to focus on the concepts of the endpoint article?

If so, why is the main focus not on its core idea? e.g.:
- What are its purposes and benefits?
- Why should I use the endpoint?
- Benefit of separating the definition of endpoint from its implementation?
- Doc and Client generation
- and some other important topics that can convey the importance of this data type

Reply:
This section outlines the ideas behind zio http.. 

using this as a guideline https://diataxis.fr/explanation/

Reply:
This show more in depth to it.. https://github.com/zio/zio-http/blob/main/docs/dsl/endpoint.md

Reply:
@khajavi